sdk/storage/azure-storage-files-shares/src/share_file_client.cpp

```diff
@@ -346,6 +355,15 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
     {
       protocolLayerOptions.FileLastWriteTime = std::string(FileCopySourceTime);
     }
+    if (options.SmbProperties.ChangedOn.HasValue())
+    {
+      protocolLayerOptions.FileChangeTime = options.SmbProperties.ChangedOn.Value().ToString(
+          Azure::DateTime::DateFormat::Rfc3339, DateTime::TimeFractionFormat::AllDigits);
+    }
+    else
+    {
+      protocolLayerOptions.FileChangeTime = std::string(FileCopySourceTime);
```

Comment:
I think the default behavior is wrong.
According to the [REST API reference](https://docs.microsoft.com/en-us/rest/api/storageservices/copy-file), the default behavior is the time when the copy operation is initiated.

Anyway, let's just remove this `else` clause and let service decide the default behavior,

Reply:
Yeah, I'll fix it.